sbt-web-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/WebScalaJSBundlerPlugin.scala

```diff
@@ -60,19 +60,25 @@ object WebScalaJSBundlerPlugin extends AutoPlugin {
     )
   }
 
-  def bundlesWithSourceMaps(sjsStage: TaskKey[Attributed[File]]): Def.Initialize[Task[Seq[(File, String)]]] =
+  val bundlesWithSourceMaps: Def.Initialize[Task[Seq[(File, String)]]] =
     Def.taskDyn {
       val projects = scalaJSProjects.value.map(Project.projectToRef)
       Def.task {
         // ((file, relative-path), sourceMapsEnabled)
         val bundles: Seq[((File, String), Boolean)] =
           projects
             .map { project =>
-              Def.task {
-                val files = (webpack in (project, Compile, sjsStage in project)).value
-                val clientTarget = (npmUpdate in (project, Compile)).value
-                val sourceMapsEnabled = (finallyEmitSourceMaps in (project, Compile, sjsStage in project)).value
-                files.map(_.data).pair(Path.relativeTo(clientTarget)).map((_, sourceMapsEnabled))
+              Def.taskDyn {
+                val sjsStage = (scalaJSStage in project).value match {
+                  case Stage.FastOpt => fastOptJS
+                  case Stage.FullOpt => fullOptJS
+                }
+                Def.task {
+                  val files = (webpack in(project, Compile, sjsStage in project)).value
```

Comment:
I don't think the `in project` achieves anything here. Also, please keep the space after the `in` (also below):
```suggestion
                  val files = (webpack in (project, Compile, sjsStage)).value
```